pkgs/development/compilers/swift/swiftpm/default.nix

```diff
@@ -385,6 +389,7 @@ in stdenv.mkDerivation (commonAttrs // {
     swiftpmMakeMutable swift-tools-support-core
     substituteInPlace .build/checkouts/swift-tools-support-core/Sources/TSCTestSupport/XCTestCasePerf.swift \
       --replace 'canImport(Darwin)' 'false'
+    patch -p1 -d .build/checkouts/swift-tools-support-core -i ${./patches/force-unwrap-file-handles.patch}
```

Comment:
Can it be converted to `patchFlags` (https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#patchflags-var-stdenv-patchflags)?

Reply:
Oh, possibly not because `configurePhase` is too late.

Reply:
I just follow the manual guidelines: https://nixos.org/manual/nixpkgs/unstable/#ssec-swiftpm-patching-dependencies. There is a configure phase inside the helper function: https://github.com/NixOS/nixpkgs/blob/00000001b4f4a57961f35ba1e0db5a4806a38ba2/pkgs/development/tools/swiftpm2nix/support.nix#L35